### ISSUE TITLE
[Vue] [RadListView] Scroll to Index on iOS won't work until a manual scroll is done

### ISSUE THREAD

**Issue by hazzo** [telerik/nativescript-ui-feedback#885]
### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
When trying to scrollToIndex on iOS (with staggered layout) it won't work until you perform a manual scroll and then execute the `scrollToIndex` method.

![](https://media.giphy.com/media/1BdsG8okAtqVD0KPT2/giphy.gif)

### Which platform(s) does your issue occur on?
iOS 12.0

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:**  "nativescript-ui-listview": "^3.7.1",
- **CLI:** 4.2.4
- **Cross-platform modules:** ^4.2.0
- **Runtime(s):** 4.2.0

### Please tell us how to recreate the issue in as much detail as possible.
- Create a RadListView with staggered layout
- Try to `scrollToIndex` on load, *won't work*
- Scroll to bottom and then to top manually
- Execute `scrollToIndex` with tap action, *it works*.
- Once you scrolled up and down `scrollToIndex` will now always work.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

Template
```html
<template>
    <Page>
        <MainActionBar/>
        <AbsoluteLayout>
            <RadListView ref="listView"
                         for="item in itemList"
                         class="chatlist__list"
                         layout="staggered"
                         gridSpanCount="1"
                         @itemTap="onItemTap"
                         @loaded="onLoaded"
            >
                <v-template>
                    <FlexboxLayout class="chatlist__list__element" flexDirection="column">
                        <Label :text="item.name"></Label>
                        <Label :textWrap="true" :text="item.description"></Label>
                    </FlexboxLayout>
                </v-template>
            </RadListView>
            <FlexboxLayout class="chatlist__fab" alignItems="center" justifyContent="center" @tap="onLoaded">
                <Label class="zmdi zmdi-24" :text="'zmdi-plus' | fonticon"/>
            </FlexboxLayout>
        </AbsoluteLayout>
    </Page>
</template>
```
Javascript
```javascript
    export default {
        data() {
            return {
                itemList: [
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description Item 3 description Item 3 description Item 3 description Item 3 description Item 3 description Item 3 description Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                    {name: 'Item 1', description: 'Item 1 description'},
                    {name: 'Item 2', description: 'Item 2 description'},
                    {name: 'Item 3', description: 'Item 3 description'},
                ],
            }
        },

        methods: {
            onItemTap({item}) {
                console.log(`Tapped on ${item.name}`);
            },

            onLoaded () {
                setTimeout(() => {
                    console.log('test')
                    this.$refs.listView.scrollToIndex(21, true)
                }, 500);
            },
        },
    }
```

**Comment by NickIliev** [comment]
@hazzo the method `scrollToIndex` is working as expected - in your case the items are not enough so that it can be scrolled (or at least with the default styles they are all visible on the device screen).

Demo app demonstrating the above case with additional items and margins can be found [here](https://github.com/NickIliev/NS-Issues-2018-II/tree/master/nativescript-ui-feedback/issue-885-vue)

**Comment by hazzo** [comment]
Thanks @Nicklliev. I will try your example and if works I'll try to isolate what's causing the issue in my project.

**Comment by hazzo** [comment]
Hi @NickIliev again, I was able to discover what was the issue. For some reason iOS needs a minimum amount of elements to make the scroll to index (even though there is a minimum amount of elements to have a scroll area). At first I though it was that the elements should have a minimum height, but I removed every style on it and increased the item count and the scroll to index also worked. Really strange. Thanks for the demo!
